pkg/controller/clusterdeprovision/clusterdeprovision_controller.go

```diff
@@ -201,6 +216,50 @@ func (r *ReconcileClusterDeprovision) Reconcile(request reconcile.Request) (reco
 		return reconcile.Result{}, nil
 	}
 
+	actuator := r.getActuator(instance)
+	if actuator != nil {
+		// actuator found, ensure creds work.
+		err := actuator.TestCredentials(instance, r.Client, rLog)
+		if err != nil {
+			rLog.WithError(err).Error("Credential check failed")
+
+			conditions, changed := controllerutils.SetClusterDeprovisionConditionWithChangeCheck(
+				instance.Status.Conditions,
+				hivev1.AuthenticationFailureClusterDeprovisionCondition,
+				corev1.ConditionTrue,
+				authenticationFailedReason,
+				err.Error(),
```

Comment:
These error messages don't contain any request IDs or anything that could change each time through? Just want to be sure we don't hotloop here.